config/config-template-zap-mac.yaml

```diff
@@ -32,11 +32,12 @@ general:
     #   podman: RapiDAST runs each scanner using podman
     #   flatpak: RapiDAST runs each scanner using flatpak
     #   none: RapiDAST runs each scanner in the same host or container (where RapiDAST itself is running in a container)
-    type: "none"
+    # When undefined, relies on rapidast-defaults.yaml, or `podman` if nothing is set
+    #type: "none"
```

Comment:
can we leave this without change for Mac config only? as for MacOS, we'd prefer 'none' to 'podman'.